### PR TITLE
Add MeritooCommonBundle recipe

### PR DESCRIPTION
**Pull request by meritoo**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by Nyholm** [comment]
This looks good. Can you have a look why Travis is failing though?

File: meritoo/common-bundle/0.1/config/packages/meritoo_common.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#
+# All configuration options are described here:
+# https://github.com/meritoo/common-bundle/blob/master/docs/Configuration.md
```
Comment (OskarStark) [inline meritoo/common-bundle/0.1/config/packages/meritoo_common.yaml]: the could be a one liner

Comment (meritoo) [reply meritoo/common-bundle/0.1/config/packages/meritoo_common.yaml]: In this case line will be longer than 120, so it's better to use 2 lines

**Comment by Nyholm** [comment]
Did you manage to find out why sf3.4 is failing?

**Comment by meritoo** [comment]
As mentioned in https://github.com/symfony/symfony/issues/28062 boolean value will not work properly with Symfony 3.4. Travis runs 2 jobs, 1 per Symfony version:
* 3.4
* 4.0

Is there possibility to use separate `manifest.json` per each version? How to solve this issue?

**Comment by meritoo** [comment]
Hi everyone. Have you any ideas how to solve this issue 👉 https://github.com/symfony/recipes-contrib/pull/425#issuecomment-407869190? 🙃

**Comment by meritoo** [comment]
@Nyholm Thank you 🙂

**Pull request merged**